pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -217,21 +178,57 @@ func (mattermost *ClusterInstallation) GenerateService(serviceName, selectorName
 					Kind:    "ClusterInstallation",
 				}),
 			},
-			Annotations: svcAnnotations,
+			Annotations: annotations,
 		},
 		Spec: corev1.ServiceSpec{
-			Ports: []corev1.ServicePort{
-				{
-					Port:       8065,
-					TargetPort: intstr.FromString("app"),
-				},
-			},
-			Selector:  ClusterInstallationLabels(selectorName),
-			ClusterIP: corev1.ClusterIPNone,
+			Selector: ClusterInstallationLabels(selectorName),
 		},
 	}
 }
 
+// GenerateService returns the service for Mattermost
```

Comment:
nit: to match other comment style.
```suggestion
// GenerateService returns the service for Mattermost.
```

Reply:
Yes, sure - missed a dot